tests/e2e/smoketest-simple/00-install-minio.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: "kubectl create --namespace $NAMESPACE -f ../minio.yaml"
```

Comment:
Just a question: would it make sense to move this logic to a `make` rule? I mean: if this is used just for the E2E tests, is fine. But if we expect people to run their "hello world" deploying `minio` first, probably it makes sense to add a `make` rule to make everything easier

Reply:
Maybe?  I'm not really sure what will be done longterm.  If we end up adding a Makefile just for tests it could certainly go there.  I also need to look into whether you can deploy a single minio instance that can be used cluster wide.

Reply:
It also would give us some flexibility on the storage side. We could use something like `../minio_losing_blocks.yaml` while testing for the compactor. (but maybe thats out of scope)

Reply:
@frzifus  after chatting with @iblancasa I've added a make target and am leaving it so that minio is installed in the namespace for each test.  The minio install is quite fast so it shouldn't be a problem.